File: market-data-public/markets-operational/markets-operational.js
const axios = require('axios');

async function fetchOperationalMarketData(marketCode) {
  const url = `https://api.ox.fun/v3/markets/operational?marketCode=${marketCode}`;

  try {
    const response = await axios.get(url);
    console.log('Operational Market Data:', response.data);
  } catch (error) {
    console.error('Error fetching operational market data:', error.response ? error.response.data : error.message);
  }
}

// Example usage
fetchOperationalMarketData('BTC-USD-SWAP-LIN');

File: Rest-Api-V3/market-data-public/markets-operational/markets-operational.py
import requests

def fetch_operational_market_data(market_code):
    url = f'https://api.ox.fun/v3/markets/operational?marketCode={market_code}'

    try:
        response = requests.get(url)
        response.raise_for_status()  # Raise an error for bad responses
        print('Operational Market Data:', response.json())
    except requests.exceptions.RequestException as error:
        print('Error fetching operational market data:', error)

# Example usage
fetch_operational_market_data('BTC-USD-SWAP-LIN')